src/ExpressionEvaluator/Core/Source/ExpressionCompiler/ILSpan.cs

```diff
@@ -7,7 +7,7 @@
 namespace Microsoft.CodeAnalysis.ExpressionEvaluator
 {
     // TODO: use TextSpan?
-    public struct ILSpan : IEquatable<ILSpan>
+    internal struct ILSpan : IEquatable<ILSpan>
```

Comment:
Looks like we don't run it for ExpressionCompiler. That's likely how this got unnoticed in the first place. Will follow up with adding the public api files.